crate/azure_cvm/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "azure_cvm"
+version.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+
+[dependencies]
+base64 = "0.21"
```

Comment:
some of these dependencies are already declared at workspace level. Probably some of them can be "mutualized" with `maa_client` deps.
You can naturally post-pone this remark since the merge is  urgent

Reply:
Fixed in 4767391487ed7165d356b439f270d4b3cf1d3cdc.